test-omni-integ-regtest.sh

```diff
@@ -9,7 +9,7 @@ trap cleanup EXIT
 BTCD=copied-artifacts/src/omnicored
 DATADIR=$HOME/.bitcoin
 LOGDIR=logs
-MSCLOG=/tmp/mastercore.log
+MSCLOG=/tmp/omnicore.log
```

Comment:
I think I added the symlink so the log could be viewed in the workspace [here](https://ci.omni.foundation/job/omni-integ-regtest/ws/logs/) -- I can't even remember. We can probably remove that variable and the symlink.